.ci/check_man.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+if [ $CIRCLECI ]; then
+    sudo apt install man
+    pagepath=~/.local/man/man1/coala.1
+elif [ $TRAVIS ]; then
+    pagepath=$VIRTUAL_ENV/man/man1/coala.1
+else
+    pagepath=coala
+fi
+
+man $pagepath | grep "https://coala.io" > /dev/null 2>&1
```

Comment:
to know it is installed correctly, the command needs to be `man coala` without an explicit path, or a `.1` suffix.

If that isnt possible, I guess setting `MANPATH` is a bit better, but then we need to tell users to also set MANPATH, and that gets complicated when they have different OS, venvs, etc.